bin/build.js

```diff
@@ -64,26 +63,9 @@ if (argv.w) {
   buildOptions.configTransformer = transformFunction;
 }
 
-const BREAKOUT = 'compilation_error';
-
 build(buildOptions)
-  .then((stats) => {
-    console.log('Webpacking compilation result:\n', stats.toString({
-      colors: !!supportsColor.stdout,
-      // hide excessive chunking output
-      chunks: false,
-      // hide other built modules
-      maxModules: 0,
-      // hide warning traces
-      moduleTrace: false
-    }));
-
-    if (stats.hasErrors()) {
-      throw new Error(BREAKOUT);
-    }
-  })
   .catch((error) => {
-    if (error.message === BREAKOUT) {
+    if (error.message === 'compilation_error') {
```

Comment:
Did you mean to convert the old constant to a string literal? It seems like there is still logic in other places in the code that use this string and a constant would still be nice

Reply:
The constant got split across files and I felt like the string was almost equivalent to use a shared constant.